packages/map/src/leaflet/GMap.ts

```diff
@@ -26,8 +26,15 @@ export class GMapLayer extends TileLayer {
     layerEnter(map: Map) {
         super.layerEnter(map);
         this.add(new (GoogleMutant as any)({
-            type: "roadmap"
+            type: "roadmap",
+            styles: this.googleMapStyles()
         }));
     }
 }
 GMapLayer.prototype._class += " map_MapBoxLayer";
+export interface GMapLayer {
+    googleMapStyles(): object;
+    googleMapStyles(_: object): this;
+}
+
+GMapLayer.prototype.publish("googleMapStyles", {}, "object", "Styling for map colors etc", null, { tags: ["Basic"] });
```

Comment:
This doesn't allow the user to change the styles on the fly in the property editor (only works before first render)?

Reply:
Right, the change to add this feature "on-enter-only" is very minimal and provides a great deal of customizability. I don't think GoogleMutant exposes the capability of modifying the `styles` parameter post-render

Reply:
We could tweak (improve) the following code to force a reload of GMap:  https://github.com/hpcc-systems/Visualization/blob/master/packages/map/src/leaflet/Leaflet.ts#L146-L148

Reply:
I'm not sure what those lines are doing. Does that conditional essentially mean "if the map needs to be reloaded"?